scripts/nb-tester/test-notebook.py

```diff
@@ -35,6 +35,36 @@
 ]
 
 
+def filter_paths(paths: list[Path], submit_jobs: bool) -> list[Path]:
```

Comment:
Nicer is to return `Iterator[Path]`. Every time you want to return a path, you will `yield path`. No need to have `good_paths` as a collection - remove that. No return statement.

https://github.com/Qiskit/documentation/blob/ff3500a3926b4b9663dee35adf007b0e036cd34a/start#L21-L29

Reply:
Much nicer, thanks!